flows/runs/event_log.go

```diff
@@ -0,0 +1,56 @@
+package runs
+
+import (
+	"encoding/json"
+
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/utils"
+)
+
+type eventLogEntry struct {
```

Comment:
This naming seems a bit tortured. It is a log, of events, but actually no it is a log of entries which are of events? 

What about it is just a log and it has log entries? We don't need to describe that the log entries are of events do we?